### PR TITLE
Feature/sleep dom

### PR DESCRIPTION
**Pull request by catlynbowles**
What does this PR do?
- Adds sleep data to DOM. 

Where should your reviewer start?
- index.html line 38
- Hydration.js line 25
- Sleep.js line 60
- scripts.js line 5

What (if any) features are you implementing?
- Display weekly data for sleep and hydration, and friend names. 

What (if anything) did you refactor?
- Refactor Hydration method to remove date parse. 

Were there any issues that arose?
- Yes, difficulties with date parse. 

Is there anything that you need from your teammate?
- N/A

Any other comments, questions, or concerns?
- N/A

### CODE REVIEW

File: src/Hydration.js [hunk 1]
```diff
@@ -25,43 +24,41 @@ class Hydration {
...
-
-    let mostRecentDayData = parsedDates.reduce((highestNum, user) => {
-      if (highestNum.date > user.date) {
-        return highestNum;
-      }
-      return user;
-    });
-    // console.log('numOZ', mostRecentDayData.numOunces);
-    return mostRecentDayData.numOunces;
+    let lastIndexNum = [waterData.length - 1]
```
Comment (irmakerdem) [inline src/Hydration.js:27]: Nice removing the need to parse the date! 🥳

File: src/Hydration.js [hunk 2]
```diff
@@ -25,43 +24,41 @@ class Hydration {
...
-    var actualOunces = slicedArray.map(dayObj => {
-      return dayObj.numOunces;
-    });
-    // console.log('finalOZ', actualOunces)
-    return actualOunces;
+    var final2 = final.map(ele => {
+      return JSON.stringify(ele)
+    })
+    var final3 = final2.map(ele => {
+      return ele.replace(/[{}]/g,'')
```
Comment (irmakerdem) [inline src/Hydration.js:53]: Removing curlies, I like it! 🪱

File: src/Hydration.js (hunk 1 above)
Comment (ASands17) [reply src/Hydration.js:27]: Great work on the refactor for this method!

File: src/Hydration.js [hunk 3]
```diff
@@ -25,43 +24,41 @@ class Hydration {
...
+    })
+    var final4 = final3.map(ele => {
+      return ele.replace(/['"]/g,'')
```
Comment (irmakerdem) [inline src/Hydration.js:56]: Removing quotes, nice! 🗣

File: src/Sleep.js [hunk 4]
```diff
@@ -57,6 +57,17 @@ class Sleep{
     let final = wholeWeek.map(day => {
       return {[day.date]: day.hoursSlept}
     });
+    
+    var final2 = final.map(ele => {
+      return JSON.stringify(ele)
```
Comment (irmakerdem) [inline src/Sleep.js:62]: Making strings!! 🪢

**Review by ASands17 — COMMENTED** [review]
Great work on this. We may change how we display things on the DOM in a different interation, but this works great for now.

**Review by irmakerdem — APPROVED** [review]
Looks good!

**Pull request merged**